code/_onclick/hud/alert.dm

```diff
@@ -177,6 +177,41 @@
 	desc = "You're severely malnourished. The hunger pains make moving around a chore."
 	icon_state = "starving"
 
+///Vampire "hunger"
+
+/obj/screen/alert/fat/vampire
+	name = "Fat"
+	desc = "You drank too much blood, somehow, lardass. Run around the station and lose some weight."
+	icon_state = "v_fat"
+
+/obj/screen/alert/full/vampire
+	name = "Full"
+	desc = "You feel full and satisfied, but you know you will thirst for more blood soon..."
+	icon_state = "v_full"
+
+/obj/screen/alert/well_fed/vampire
+	name = "Well Fed"
+	desc = "You feel quite satisfied, but you could do with a bit more blood."
+	icon_state = "v_well_fed"
+
+/obj/screen/alert/fed/vampire
+	name = "Fed"
+	desc = "You feel moderately satisfied, but a bit more of blood may not hurt."
```

Comment:
"You feel moderately satisfied, but a bit more blood wouldn't hurt." reads a bit less robotically.

Reply:
Updated to reflect suggestion.